### ISSUE TITLE
remove_binary Option Missing for Attachment

### ISSUE THREAD

**Issue by SoccerField24x7** [elastic/elasticsearch-net#7961]
I am trying to create an ingest pipeline for attachments and there doesn't appear to be a way to set the `remove_binary` option. I would like to do something like:

```
var ingestPipelineResult = await client.Ingest.PutPipelineAsync("attachment", p => 
    p.Processors(x =>
    {
        x.Attachment<Document>(a =>
        {
            a.Field(f => f.Data);
            a.IndexedChars(-1);
            a.RemoveBinary(true); // this is what I want; not a real thing
        });
    }));
```

I suppose I could re-put the document after removing the `data` property, but that seems like overkill when the functionality exists in Elasticsearch. I could also try my hand at creating a raw request using something like `HttpTransportClient`, but I haven't seen any examples of how this would work so it will take some trial and error.

**Comment by flobernd** [comment]
Hi  @SoccerField24x7, it seems like this field is missing from the specification and should be added there. I'll take care!

**Comment by flobernd** [comment]
This is fixed in 8.13.x.
